doc/index.rst

```diff
@@ -29,6 +29,10 @@ This system is what was previously called m&m, which is no longer available. m&m
 Installing
 -----------------------------
 
+Download the system you want to use. If you use classflexing, the Allclass system will enable everything but it has it's kinks. If you plan on sticking to one class, the class system will be the one you download. DO NOT EXTRACT THE SYSTEM. After downloading, open Mudlet and your profile and open the Package Manager. Click install and select the .zip system that you downloaded. It will ask you to select where the m&m file is, you'll want to choose where your profile folder is saved as the file will be there. 
```

Comment:
I'll try to add that for the next release - I'll get this one out for the latest batch of envoy changes